presto-product-tests/conf/docker/singlenode-kerberos-kms-hdfs-no-impersonation/docker-compose.yml

```diff
@@ -1,17 +0,0 @@
-version: '2'
-services:
-
-  hadoop-master:
-    image: 'prestodev/cdh5.15-hive-kerberized-kms:${DOCKER_IMAGES_VERSION}'
-    volumes:
-    - ../../../conf/hadoop/singlenode-kerberos-kms-hdfs-impersonation/kms-core-site.xml:/etc/hadoop-kms/conf/core-site.xml
```

Comment:
It would be better to un-share these config files in a separate commit.

Reply:
I agree. but the benefit is minimal, so let me not redo the history now.